src/main/java/com/pff/PSTByteFileContent.java

```diff
@@ -0,0 +1,43 @@
+package com.pff;
+
+public class PSTByteFileContent extends PSTFileContent{
+    
+    protected byte[] content;
+    protected int index;
+    
+    public PSTByteFileContent(byte[] content){
+        this.content = content;
+        this.index = 0;
+    }
+    
+    public byte[] getBytes(){
+        return this.content;
+    }
+    public void setBytes(byte[] content){
+        this.content = content;
+    }
+    
+    public void seek(long index){
+        this.index = (int) index;
+    }
+    public long getFilePointer(){
+        return this.index;
+    }
+    public int read(){
+        return (this.index >= this.content.length) ? -1 : (int) this.content[index++];
+    }
+    public int read(byte[] target){
+        if(this.index >= this.content.length) return -1;
+        int targetindex = 0;
+        while(targetindex < target.length & this.index < this.content.length){
```

Comment:
Nope. `&&` is branching whereas `&` is not, that means it evaluates faster. `&&` is best for circumstances like `if(expensiveMethodA() && expensiveMethodB())` or `if(var != null && var.getSomething() == somethingElse)`. In pretty much everything else `&` and `|` are more performant.